docs/guides/universal-profile/browser-extension/03-interact-with-dapp.md

```diff
@@ -9,173 +9,187 @@ import TabItem from '@theme/TabItem';
+The extension injects an `ethereum` object into a web page as `window.ethereum` to interact with the extension.
+
+### 1. Initialize a blockchain provider.
 
 <Tabs>
   <TabItem value="web3" label="web3">
 
-  ```js
-  import Web3 from 'web3';
-  const web3 = new Web3(window.ethereum);
-  ```
+```js
+import Web3 from 'web3';
+const web3 = new Web3(window.ethereum);
+```
 
   </TabItem>
   <TabItem value="etherjs" label="etherjs">
 
-  ```js
-  import { ethers } from 'ethers';
-  const etherProvider = new ethers.providers.Web3Provider(window.ethereum);
-  ```
+```js
+import { ethers } from 'ethers';
+const etherProvider = new ethers.providers.Web3Provider(window.ethereum);
+```
 
   </TabItem>
 </Tabs>
 
-
-  
-#### Get an access to the UP address the user is controlling
+### 2. Get access to the UP address in use.
 
 A call to `requestAccounts` will open an extension popup to authorize an account.
 
 <Tabs>
   <TabItem value="web3" label="web3">
 
-  ```js
-  const accountsRequest: string[] = await web3.eth.requestAccounts();
-  const accounts: string[] = await web3.eth.getAccounts(); //should also yield the same address
-  ```
+```js
+const accountsRequest: string[] = await web3.eth.requestAccounts();
+const accounts: string[] = await web3.eth.getAccounts(); //should also yield the same address
+```
 
   </TabItem>
   <TabItem value="etherjs" label="etherjs">
 
-  ```js
-  const accountsRequest: string[] = await etherProvider.send('eth_requestAccounts', []);
-  const signer = etherProvider.getSigner();
-  await signer.getAddress(); //should also yield the same address
-  ```
+```js
+const accountsRequest: string[] = await etherProvider.send(
+  'eth_requestAccounts',
+  [],
+);
+const signer = etherProvider.getSigner();
+await signer.getAddress(); //should also yield the same address
+```
 
   </TabItem>
   <TabItem value="raw" label="raw">
 
-  ```js
-  const accountsRequest: string[] = await window.ethereum.request({method:'eth_requestAccounts', params:[]});
-  const accounts: string[] = await window.ethereum.request({method:'eth_accounts', params:[]}); //should also yield the same address
-  ```
+```js
+const accountsRequest: string[] = await window.ethereum.request({
+  method: 'eth_requestAccounts',
+  params: [],
+});
+const accounts: string[] = await window.ethereum.request({
+  method: 'eth_accounts',
+  params: [],
+}); //should also yield the same address
+```
 
   </TabItem>
 </Tabs>
 
-### Operations
+## Operations
 
-Each operation will open an extension popup window for a user to confirm the transaction / sign the message.
+Each operation will open an extension popup window for a user to confirm the transaction or sign the message.
 
 :::note
-If you don't have funds on your EOA address, the relayer service will be used to pay the gas fees (thus ignoring gas properties).
+If you don't have funds at your EOA address, the application will use the relayer <br/> service by default to pay the gas fees (thus ignoring gas properties).
```

Comment:
at? Why not "on" ?

Not sure if the `<br/>` is necessary here? Why add line break here?

Reply:
`<br>` > `<br/>`

Reply:
Guess this was a plain Grammarly issue thinking about an IRL address. The line break was just for the looks, but you´re right: Its not necessary here because of the added :::note:::